apps/bare-expo/ios/BareExpo/Info.plist

```diff
@@ -96,20 +98,5 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
-	<key>UIRequiresFullScreen</key>
-	<false/>
-	<key>LSApplicationQueriesSchemes</key>
-	<array>
-		<string>fbapi</string>
-		<string>fb-messenger-api</string>
-		<string>fbauth2</string>
-		<string>fbshareextension</string>
-	</array>
-	<key>CFBundleDisplayName</key>
-	<string>BareExpo</string>
-	<key>FacebookAppID</key>
-	<string>629712900716487</string>
-	<key>FacebookDisplayName</key>
-	<string>Expo APIs</string>
```

Comment:
Changed it back in https://github.com/expo/expo/pull/14512/commits/6fb0dd0f59aa03128fe4df08491bece77a879f87. I think I changed it because I had some trouble with building and forgot to change it back.